migtests/tests/oracle/ssl/cleanup-db

```diff
@@ -6,4 +6,10 @@ set -x
 source ${SCRIPTS}/functions.sh
 
 echo "Deleting existing data in source database"
-run_sqlplus_as_schema_owner ${SOURCE_DB_NAME} ${TESTS_DIR}/oracle/utils/delete_full_schema
\ No newline at end of file
+run_sqlplus_as_schema_owner ${SOURCE_DB_NAME} ${TESTS_DIR}/oracle/utils/delete_full_schema
+
+if [ "${SOURCE_REPLICA_DB_SCHEMA}" != "" ]
+then
+	echo "Deleting existing data in FF database"
+	run_sqlplus ${SOURCE_DB_NAME} ${SOURCE_REPLICA_DB_SCHEMA} ${SOURCE_REPLICA_DB_PASSWORD} ${TESTS_DIR}/oracle/utils/delete_full_schema
```

Comment:
We don't need to `run_sqlplus_as_schema_owner` ?

Reply:
Just copied it from the other tests. Also checked it in the automation. Seems to be working fine without superuser.